src/tools/gcc.lua

```diff
@@ -306,6 +306,8 @@
 		if table.contains(os.getSystemTags(cfg.system), "darwin") then
 			if cfg.sharedlibtype == "OSXBundle" then
 				return "-bundle"
+			elseif cfg.sharedlibtype == "XCTest" then
```

Comment:
This probably could have been an `or` on the previous line, but the consistency is good. :+1: